### DISCUSSION TITLE
Web site not displaying log

### DISCUSSION THREAD

**Discussion by marco-russian** [Flexget/Flexget#3276] [Q&A] [answered]
Hi,

Recently I have noticed that the log section of the web UI is not displaying the log. I can verify that the log file is being populated. I'm running flexget on a raspberry PI on the latest OS build, Python 3.7 and the latest release of Flexget.

Anyone else experiencing this issue?
[2021-12-09_09-09-59.pdf](https://github.com/Flexget/Flexget/files/7681032/2021-12-09_09-09-59.pdf)

**Comment by jswclw** [comment]
I'm seeing the same - log file is populated, but web interface is not showing anything. It indicates that it is streaming, but nothing. FlexGet 3.2.7.

**Comment by superjerrry** [comment]
Latest version 3.2.8 same problem dont show log

**Comment by gazpachoking** [comment, marked as answer]
Hmm, looks like this is a bug in the pyparsing library. https://github.com/pyparsing/pyparsing/issues/350 Hopefully they cut a release soon. In the mean time I've reverted our version, so our next release should fix this issue.
EDIT: new release with the fix is out

**Reply by shotfirer** [reply]
Sorry for hijacking the old thread, but I'm running v3.3.8 and I have exactly the same issue as OP, I don't see the log in the web UI. It started with version 3.2.something and never worked for me ever since. No errors in the log nor in the Chrome console.
I'm running FlexGet in daemon mode on OrangePi under Armbian 21.02.3 and Python 3.8.10. Pyparsing v 3.0.7

**Reply by jswclw** [reply]
Looks like reverting pyparsing to 2.4.7 is still required.

**Reply by Arturoe1** [reply]
Yes the error is back, it worked fine for a while but it doesn't work anymore.
